src/internal/operations/backup.go

```diff
@@ -154,11 +163,139 @@ func (op *BackupOperation) Run(ctx context.Context) (err error) {
 	return err
 }
 
+// calls kopia to retrieve prior backup manifests, metadata collections to supply backup heuristics.
+func produceManifestsAndMetadata(
+	ctx context.Context,
+	kw *kopia.Wrapper,
+	sw *store.Wrapper,
+	sel selectors.Selector,
+	acct account.Account,
+) ([]*snapshot.Manifest, []data.Collection, error) {
+	complete, closer := observe.MessageWithCompletion("Fetching backup heuristics:")
+	defer func() {
+		complete <- struct{}{}
+		close(complete)
+		closer()
+	}()
+
+	m365, err := acct.M365Config()
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var (
+		tid         = m365.AzureTenantID
+		oc          = selectorToOwnersCats(sel)
+		ms          = kopia.FetchPrevSnapshotManifests(ctx, kw.Conn(), &oc)
```

Comment:
I am going to introduce another snapshot to store backup details. We'll need to make sure that gets filtered out here.

Reply:
if we add a tag to it that somehow indicates it's for backup details it should be easy to filter out. Or perhaps don't add the same tags we do for regular backups and then it won't be returned when we search for snapshots that contain item data. It just depends on how we plan to lookup backup details snapshots really

Reply:
Well - will probably add some bookkeeping tags TBD - but I think the logic in this method should include only the relevant snapshots vs trying to filter out specific tags.

Reply:
no tags would work just fine. It's just that looking up things with tags in kopia is an "includes at least these tags" operator not an "exactly these tags" operator. So if the backup details snapshot had a superset of the tags we add to snapshots with item data then we'd need to filter those snapshots out ourselves

Reply:
Going to update this PR to pass in multiple tags to `BackupCollections`.  One being the backupID, another as a categorical identifier to separate backups from details.  That'll get us a lookup that's sufficient for this change, and we can add tagging to details in the future as needed.